revolutionpi/gpio_chip.go

```diff
@@ -120,13 +148,11 @@ func (g *gpioChip) getBitValue(address int64, bitPosition uint8) (bool, error) {
 func (g *gpioChip) writeValue(address int64, b []byte) error {
 	g.logger.Debugf("Writing %#d to %v", b, address)
 	n, err := g.fileHandle.WriteAt(b, address)
-	g.logger.Debugf("Wrote %#d byte(s), n: %d", b, n)
 	if err != nil {
 		return err
 	}
-	if n < 1 || n > 1 {
-		return fmt.Errorf("expected 1 byte(s), got %#v", b)
-	}
```

Comment:
removed this logic because it was unnecessary, and allows writeValue to be used with multiple write apis